config/initializers/email_prefixer.rb

```diff
@@ -1,3 +1,3 @@
 EmailPrefixer.configure do |config|
-  config.stage_name = 'WHEEL'
+  config.stage_name = Rails.application.secrets.subject_prefix_for_outgoing_emails
```

Comment:
@mohitnatoo done. I added env name also. If env name is `production` then this gem does not display env name which is what we want.